docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -74,7 +79,8 @@ You can initialize a selector with any of these objects.
     import { Selector } from 'testcafe';
 
     // A selector is created from a CSS selector string.
-    // This selector will return the first matching DOM node.
+    // This selector will return the first matching DOM node
+    // when you pass it to an action or an assertion or execute it.
```

Comment:
// This selector will return the first matching DOM node.
// You can execute this selector or pass it as a parameter to an action or assertion.